promscale/installation/kubernetes/helm.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Install Promscale on Kubernetes using Helm
+excerpt: Install Promscale on a Kubernetes cluster using Helm
+product: promscale
+keywords: [Kubernetes, analytics, Helm]
+tags: [install]
+related_pages: 
+  - /promscale/:currentVersion:/installation/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+
+# Install Promscale with Helm charts
+You can install Promscale using Helm charts and the Helm charts must be installed
+in this order:
+1. Install the TimescaleDB Helm chart
+1. Install the Promscale Helm chart
+
+## Before you begin
+1.  Understand the [resource requirements][resource-requirements] for your cluster.
+1.  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
+1.  Install [Helm 3][helm] in your local machine.
+1.  Install [kubectl][kubectl] in your local environment and
```

Comment:
```suggestion
1.  Understand the [resource requirements][resource-requirements] for your cluster.
1.  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
1.  Install [Helm 3][helm] in your local machine.
1.  Install [kubectl][kubectl] in your local environment and
```

Is order important here? If not, make them bullets.

Reply:
Yes, I wanted it as a bullet list. Let me rework some steps again.

Reply:
@Loquacity steps 4 onwards need to be in order so I guess we need to keep it as ordered and I agree that this is not consistent with our other Before you begin that have paras or bullets.

Reply:
What if we separate it into two lists with their own stem sentences? Could that work?

Reply:
> What if we separate it into two lists with their own stem sentences? Could that work?

makes sense!!! let me try that.